### PR TITLE
[12.0][ADD]phone_validation_e164

### PR DESCRIPTION
**Pull request by dsolanki-initos**
@jans23 FYI

### CODE REVIEW

File: phone_validation_e164/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    'name': "Nitrokey Customization",
```
Comment (jans23) [inline phone_validation_e164/__manifest__.py]: Phone Validation e164

File: phone_validation_e164/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    'description': """
+    We have added defaule e164 phone formator
```
Comment (jans23) [inline phone_validation_e164/__manifest__.py]: Proper description, please.

File: phone_validation_e164/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    """,
+    'category': 'Nitrokey',
```
Comment (jans23) [inline phone_validation_e164/__manifest__.py]: I think this is wrong.

File: phone_validation_e164/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    'version': '12.0.1.0.0',
+    'depends': [
+        'phone_validation'
+    ],
+    'author': "initOS",
```
Comment (jans23) [inline phone_validation_e164/__manifest__.py]: Also add Nitrokey GmbH

File: phone_validation_e164/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    'website': "http://www.initos.com",
+}
```
Comment (jans23) [inline phone_validation_e164/__manifest__.py:16]: change to www.nitrokey.com

File: phone_validation_e164/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{
+    'name': "Nitrokey Customization",
+    'description': """
+    We have added defaule e164 phone formator
+    """,
+    'category': 'Nitrokey',
+    'version': '12.0.1.0.0',
+    'depends': [
+        'phone_validation'
```
Comment (jans23) [inline phone_validation_e164/__manifest__.py:12]: Below phone_validation_mixin is imported. Is it required as a dependency too?

File: phone_validation_e164/models/phone_validation.py [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (jans23) [inline phone_validation_e164/models/phone_validation.py]: This is not required.

File: phone_validation_e164/models/phone_validation.py [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```
Comment (jans23) [inline phone_validation_e164/models/phone_validation.py]: This is not applicable.

File: phone_validation_e164/__init__.py [hunk 9]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (jans23) [inline phone_validation_e164/__init__.py]: Add `"license": "AGPL-3",`

File: phone_validation_e164/__manifest__.py (hunk 6 above)
Comment (dsolanki-initos) [reply phone_validation_e164/__manifest__.py:12]: Yes, it is required as we have utilized the phone validation mixin(phone.validation.mixin) model

File: phone_validation_e164/models/phone_validation.py (hunk 7 above)
Comment (dsolanki-initos) [reply phone_validation_e164/models/phone_validation.py]: it is not required for v15.0

File: phone_validation_e164/__manifest__.py (hunk 6 above)
Comment (jans23) [reply phone_validation_e164/__manifest__.py:12]: So has phone_validation_mixin to be added as dependency?

File: phone_validation_e164/models/phone_validation_mixin.py [hunk 10]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (jans23) [inline phone_validation_e164/models/phone_validation_mixin.py]: again, not required.

File: phone_validation_e164/models/phone_validation.py (hunk 7 above)
Comment (jans23) [reply phone_validation_e164/models/phone_validation.py]: Is it required for 12.0?

File: phone_validation_e164/__manifest__.py (hunk 6 above)
Comment (dsolanki-initos) [reply phone_validation_e164/__manifest__.py:12]: No, phone_validation is the module name and phone_validation_mixin(phone.validation.mixin) is a model name so we have to keep "phone_validation" as a dependency

File: phone_validation_e164/models/phone_validation_mixin.py (hunk 10 above)
Comment (dsolanki-initos) [reply phone_validation_e164/models/phone_validation_mixin.py]: I've verified in other oca modules in v12.0 and they have used but if you don't need then we can remove it shall I remove it?

Comment (jans23) [reply phone_validation_e164/models/phone_validation_mixin.py]: yes

**Pull request merged**